File: csharp/module2/challenges/challenge3.cs
public class GameHealthManager
{
    private const int HealthReductionMinTurnNumber = 10; 
    public (int player1Health, int player2Health) ReduceHealthLateGame(int player1Health, int player2Health, int turnNumber, int healthReduction)
    {
        if (turnNumber >= HealthReductionMinTurnNumber)
        {
            player1Health = Math.Max(player1Health - healthReduction, 0);
            player2Health = Math.Max(player2Health - healthReduction, 0);
        }

        return (player1Health, player2Health);
    }
}

File: javascript/module2/challenges/challenge3.js
function reduceHealthLateGame(
  player1Health,
  player2Health,
  turnNumber
) {
    const healthReduction = turnNumber * 0.1;

    player1Health -= healthReduction;
    if (player1Health < 0) {
      player1Health = 0;
    }

    player2Health -= healthReduction;
    if (player2Health < 0) {
      player2Health = 0;
    }
  }

  return [player1Health, player2Health];
}
